src/app/api/rooms/[roomId]/route.ts

```
@@ -26,3 +28,38 @@ export async function GET(
+
+export async function PATCH(request: NextRequest, { params }: { params: RoomParams }) {
+  const session = await auth();
+  try {
+    if (!session) {
+      throw new UnAuthorizedError();
+    }
+
+    const roomId = Number(params.roomId);
+
+    if (isNaN(roomId)) {
+      throw new BadRequestError('유효하지 않은 ID 형식입니다.');
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)